### ISSUE TITLE
Ask for providing the GPT-4 generated false answers for NQ/TriviaQA

### ISSUE THREAD

**Issue by voidism** [likenneth/honest_llama#1]
Hi!

Thanks for the great work! It's a really nice paper.
We have a relevant on-going project and we want to compare our method with yours on the out-of-distribution setting by NQ/TriviaQA.
I noticed that your have GPT-4 generated false answers for as the multiple choices on NQ/TriviaQA.

> "... we ask GPT-4 to generate the “most plausible sounding but false" answer to serve as an adversarial data point. ..."

Can you share this false answers dataset with us? I want to fairly compare our methods with yours, so it would be very helpful if we can use the same data for evaluation. Thanks!

p.s. Another short question is, I am wondering how much cost does it take to finetune the GPT-3 true/info judger for TruthfulQA? We want to finetune it by ourselves but we are not sure whether it's affordable. Thanks!

Best,
Yung-Sung

**Comment by likenneth** [comment]
Hey! Thanks for your interest.

Working on the dataset problem now. 

To finetune GPT-true and GPT-judge, each costs roughly $10.

**Comment by voidism** [comment]
Thanks for your quick reply! Looking forward to the release of the dataset!

**Comment by 0amp** [comment]
Modified nq_open_val is available [here](https://huggingface.co/datasets/OamPatel/iti_nq_open_val). Modified trivia_qa_val is available [here](https://huggingface.co/datasets/OamPatel/iti_trivia_qa_val).

**Comment by voidism** [comment]
Hi! Thanks for providing the datasets. I tried a simple llama-7b baseline and found that the accuracy (~60 for NQ) is higher than your baseline score (46.6 for NQ) in the paper. I am wondering how did you calculate the accuracy?

I assume that your are following the "MC1" metric the same as TruthfulQA. 
In both NQ and TriviaQA, there are many true answers + one false answers.
I computed the conditional LM scores for all the true/false answers, and if max(true_answers_scores) > false_answer_score, then the accuracy of this example should be 1.0, is it correct?

I found in your paper you mentioned "if the truthful answer ranks first, it contributes one positive", not sure whether it's exactly the same as my method above.

Another question is, TriviaQA testing set should contain 11313 examples, why your trivia_qa_val file only contains 3610 examples?

Thanks!
